CHANGELOG.md

```diff
@@ -38,6 +37,8 @@ All notable changes to this project will be documented in this file.
 -   #1917 : Add C and Fortran support for set method `add()`.
 -   #1918 : Add support for set method `clear()`.
 -   #1918 : Add support for set method `copy()`.
+-   #1753 : Add support for set method `union()`.
+-   #1754 : Add support for set method `update()`.
```

Comment:
is `update` still within the scope of this PR?

Reply:
Update and union are very similar in that `c = a.union(b)` can be written as:
```python
c = a.copy()
c.update(b)
```
which may be sufficient to put it in scope.
I rely on the `update` implementation for the case where `union` takes a non-set argument.
But I didn't actually change anything for `update` in this PR. I just activated the tests that were already working. Update is translated to multiple add nodes so it has been working since `add` was added.

If you prefer I can split this into 2 PRs?

Reply:
I don't think `update` is fully supported.
In Python, it only processes the first argument:
```
if __name__ == '__main__':
    b = {1,2}
    c = {1,3}
    d = {1,4}
    b.update(c,d)
```
gives 
```
if __name__ == "__main__":
    b = {1, 2}
    c = {1, 3}
    d = {1, 4}
    for Dummy_0000 in c:
        b.add(Dummy_0000)
```
In C and Fortran, we cannot iterate over sets so this won't work.